components/CopyCodeBlock/CopyCodeBlock.tsx

```diff
@@ -0,0 +1,37 @@
+import { useState } from "react"
+import { FaCopy, FaCheckSquare } from "react-icons/fa"
+import style from "./copyCodeBlock.module.css"
+
+interface CodeSnippet {
+	code: string
+}
+
+const CopyCodeBlock = ({ code }: CodeSnippet) => {
+	const [copyOk, setCopyOk] = useState(false)
+
+	const iconColor = copyOk ? "#0af20a" : "#ddd"
+
+	const handleClick = () => {
+		navigator.clipboard.writeText(code)
+
+		setCopyOk(true)
+		setTimeout(() => {
+			setCopyOk(false)
+		}, 500)
```

Comment:
500 ms is too fast for some people to see the change and have time to read the text. It's good interaction feedback but I think it needs to last a bit longer so people have time to see it properly.

Reply:
You're right, I've changed it to 3 seconds, but can make it longer if you think that'd be better 🙂